hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/Constants.java

```diff
@@ -456,4 +456,15 @@ private Constants() {
   public static final String FAIL_INJECT_INCONSISTENCY_PROBABILITY =
       "fs.s3a.failinject.inconsistency.probability";
 
+  public static final String MULTIPART_DOWNLOAD_PART_SIZE = "fs.s3a.multipartdownload.part-size";
+  public static final long DEFAULT_MULTIPART_DOWNLOAD_PART_SIZE = 8_000_000;
```

Comment:
Do we have any benchmarks/workloads showing how we chose this default?

Reply:
Commented at bottom. I'll update the settings to match.